cdk/lib/openApiSpec.ts

```diff
@@ -0,0 +1,59 @@
+export const openApiSpec = {
+  swagger: "2.0",
```

Comment:
don't spend time learning Swagger 2.0, there's a newer version of this specification: OpenAPI 3.0